test/e2e/fixture/fixture.go

```diff
@@ -92,7 +92,7 @@ var (
 	testsRun                map[string]bool
 	argoCDServerName        string
 	argoCDRedisHAProxyName  string
-	argoCDRedisName         string
+	ArgoCDRedisName         string
```

Comment:
I don't see any reason to have this var plublic

Reply:
That was not intended - fixing.